### PR TITLE
Support multiple WireMockConfigurationCustomizers

### LINKED ISSUE #1500: Support multiple WireMockConfigurationCustomizers
When using WireMock, extensions such as response handlers must be configured before WireMock server is started. With Spring Contract WireMock, it's possible to register extensions in a bean that implements `org.springframework.cloud.contract.wiremock.WireMockConfigurationCustomizer` . The current implementation in Spring Contract WireMock supports only a single `WireMockConfigurationCustomizer` bean. It would be easier to modularize test libraries that build upon Spring Contract WireMock if there would be a possibility to have multiple `WireMockConfigurationCustomizer` beans.

### PR DESCRIPTION
**Pull request by lhotari**
- fixes #1500

### CODE REVIEW

File: spring-cloud-contract-wiremock/src/test/java/org/springframework/cloud/contract/wiremock/AutoConfigureWireMockConfigurationMultipleCustomizerTests.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
```
Comment (marcingrzejszczak) [inline spring-cloud-contract-wiremock/src/test/java/org/springframework/cloud/contract/wiremock/AutoConfigureWireMockConfigurationMultipleCustomizerTests.java:32]: This is not necessary cause it's already provided in `@SpringBootTest`

File: spring-cloud-contract-wiremock/src/test/java/org/springframework/cloud/contract/wiremock/AutoConfigureWireMockConfigurationMultipleCustomizerTests.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+				AutoConfigureWireMockConfigurationMultipleCustomizerTests.Config.class },
+		webEnvironment = WebEnvironment.NONE)
+@AutoConfigureWireMock(port = 0)
+public class AutoConfigureWireMockConfigurationMultipleCustomizerTests {
+
+	@Autowired
+	private Config config;
+
+	@Test
+	public void bothCustomizersHaveBeenExecuted() throws Exception {
```
Comment (marcingrzejszczak) [inline spring-cloud-contract-wiremock/src/test/java/org/springframework/cloud/contract/wiremock/AutoConfigureWireMockConfigurationMultipleCustomizerTests.java:44]: I like this test :)

**Review by marcingrzejszczak — APPROVED** [review]
LGTM - I've added one small change suggestion.

File: spring-cloud-contract-wiremock/src/test/java/org/springframework/cloud/contract/wiremock/AutoConfigureWireMockConfigurationMultipleCustomizerTests.java (hunk 1 above)
Comment (lhotari) [reply spring-cloud-contract-wiremock/src/test/java/org/springframework/cloud/contract/wiremock/AutoConfigureWireMockConfigurationMultipleCustomizerTests.java:32]: I simply copied another test as a template and followed the same pattern. JUnit 4 is used for the tests so `@RunWith(SpringRunner.class)` is required. I guess JUnit 5 migration would be something to consider.

Comment (marcingrzejszczak) [reply spring-cloud-contract-wiremock/src/test/java/org/springframework/cloud/contract/wiremock/AutoConfigureWireMockConfigurationMultipleCustomizerTests.java:32]: I've forgotten that it's a junit4 test. Indeed that's required then.

**Comment by marcingrzejszczak** [comment]
Great work, thank you so much for this contribution

**Pull request merged**